.github/actions/convertIssue/parse-issue-body.js

```diff
@@ -278,11 +278,17 @@ export async function parseIssueBody(githubIssueTemplateFile, body) {
     let connect_url =
       "https://" + combinedObject.url_abbreviation + "-openpath.nrel.gov/api/";
     configObject["server"] = {
-      connectURL: connect_url,
+      connectUrl: connect_url,
       aggregate_call_auth: "user_only",
     }; //TODO check options for call + add to form?
 
-    let subgroups = combinedObject.subgroups.split(",");
+    let subgroups = combinedObject.subgroups.split(",").map(item => item.trim());
+    if (!subgroups.includes("test")) {
+      subgroups.push("test");
+    }
+    if (!subgroups.includes("default")) {
+      subgroups.push("default");
+    }
```

Comment:
@sebastianbarry this is not the desired behavior.

> If the user does not specify opcodes, we should include valid opocodes for "test" and "default". Let me do that right now for dfc-fermata,

This will add opcodes for `default` and `test` regardless of whether the user specifies an opcode list or not.